.travis.yml

```diff
@@ -36,3 +24,84 @@ cache:
   - "$HOME/.coursier/cache"
   - "$HOME/.ivy2/cache"
   - "$HOME/.sbt/boot"
+
+jobs:
+  include:
+    - &tests
```

Comment:
It is a [yaml alias](https://github.com/cyklo/Bukkit-OtherBlocks/wiki/Aliases-(advanced-YAML-usage)). The reason I used it was to use [different environment with different builds](https://docs.travis-ci.com/user/build-stages/using-yaml-aliases/), as with build stages they didn't really get matrix's down yet. The tests could be done as a matrix, but then it would need to double a test to use.

Although this messes up the travis-ci file, it removes the entire need to worry about versions and environment variables in the scripts.